src/dlangide/ui/dsourceedit.d

```diff
@@ -245,7 +271,7 @@ class SimpleDSyntaxHighlighter : SyntaxHighlighter {
         for (;;) {
             ch = nextBracket(dir, p);
             if (!ch) // no more brackets
-                return startPos;
+                break;
```

Comment:
This caused a compiler issue for me at the last return in the function: "Statement is not reachable".